test/controllers/actions/SubmittedDataRetrievalActionProviderImplSpec.scala

```diff
@@ -55,7 +56,7 @@ class SubmittedDataRetrievalActionProviderImplSpec extends AnyWordSpecLike with
 
       val underTest = new SubmittedDataRetrievalActionProviderImpl(connector, repo)
 
-      val result = underTest.loadTaskList(taxYear, fakeOptionalRequest).value.futureValue
```

Comment:
Just out-of-interest, did you replace `futureValue` with `await` as it gave you a future not completed test failure?

Reply:
Yes, the test would start failing half the time from a timeout